src/thread.cpp

```diff
@@ -42,6 +43,11 @@ namespace real_time_tools {
       printf("Thread already running");
     }
 
+    if (parameters_.cpu_dma_latency_ >= 0)
```

Comment:
Passing a negative value for `cpu_dma_latency_` is an error. The current implementation fails silence.

Check the value of `cpu_dma_latency_` and throw a value / runtime error if it's smaller than zero here please.

Reply:
yes, i should have documented this better. now i describe this also in the doc of the parameter cpu_dma_latency that a negative value means it is not going to be set. (see new pull request)

we agreed on this logic with @MaximilienNaveau @vincentberenz @luator